app/finders/conversation_finder.rb

```diff
@@ -155,9 +155,9 @@ def current_page
   end
 
   def conversations
-    @conversations = @conversations.includes(
-      :taggings, :inbox, { assignee: { avatar_attachment: [:blob] } }, { contact: { avatar_attachment: [:blob] } }, :team, :contact_inbox
-    )
+    @conversations = @conversations.includes(:taggings, { inbox: [:channel] },
```

Comment:
There's still a N+1 query

```
GET /api/v1/accounts/1/conversations?status=open&assignee_type=all&page=1&sort_by=latest
USE eager loading detected
  Message => [:sender]
  Add to your query: .includes([:sender])
Call stack
  /Users/shivammishra/Chatwoot/chatwoot/app/models/message.rb:160:in `merge_sender_attributes'
  /Users/shivammishra/Chatwoot/chatwoot/app/models/message.rb:147:in `push_event_data'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/conversations/partials/_conversation.json.jbuilder:25:in `map'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/conversations/partials/_conversation.json.jbuilder:25:in `_app_views_api_v__conversations_partials__conversation_json_jbuilder__1828115892073730114_1685640'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/accounts/conversations/index.json.jbuilder:10:in `block (3 levels) in _app_views_api_v__accounts_conversations_index_json_jbuilder__1868484514649990367_1681860'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/accounts/conversations/index.json.jbuilder:9:in `block (2 levels) in _app_views_api_v__accounts_conversations_index_json_jbuilder__1868484514649990367_1681860'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/accounts/conversations/index.json.jbuilder:8:in `block in _app_views_api_v__accounts_conversations_index_json_jbuilder__1868484514649990367_1681860'
  /Users/shivammishra/Chatwoot/chatwoot/app/views/api/v1/accounts/conversations/index.json.jbuilder:1:in `_app_views_api_v__accounts_conversations_index_json_jbuilder__1868484514649990367_1681860'
  /Users/shivammishra/Chatwoot/chatwoot/app/controllers/concerns/switch_locale.rb:24:in `set_locale'
  /Users/shivammishra/Chatwoot/chatwoot/app/controllers/concerns/switch_locale.rb:16:in `switch_locale_using_account_locale'
  /Users/shivammishra/Chatwoot/chatwoot/app/controllers/concerns/request_exception_handler.rb:11:in `handle_with_exception'
  /Users/shivammishra/Chatwoot/chatwoot/app/controllers/concerns/switch_locale.rb:24:in `set_locale'
  /Users/shivammishra/Chatwoot/chatwoot/app/controllers/concerns/switch_locale.rb:11:in `switch_locale'
```

Wasn't able to solve this event after adding the `:sender` to messages either here or in the jbuilder. @sojan-official can you give it a shot?